Fixes #4157 - Update Edinburgh (EGPH) SMR

SamLefevre opened the pull request:
Fixes #4157 

# Summary of changes

Updated Edinburgh SMR to new style

# Screenshots (if necessary)

![image](https://user-images.githubusercontent.com/64741876/159586083-fd4ca02e-a5f2-4ebd-8c33-d45ce2f31bb5.png)

GeekPro101 commented:
@SamLefevre #4118 is open - does this develop from that?
(@luke11brown )

SamLefevre commented:
> @SamLefevre #4118 is open - does this develop from that? (@luke11brown )

No, this one is made from scratch (apart from the holding points)

PLM1995 commented:
I think this looks really really good!!

Hope you don't mind me commenting a few things I've noticed, some are a bit nit-pickey so forgive me.

Visible:
- The turn-on areas between A1 and the 06 starter extention and between D1 and the 24 starter extention are shown as the same colour as the taxiways in the reference image, but you've done then runway colour here.
- The A10 and A11 hold labels don't seem to line up with the hold lines.

Implementation:
- You have used hard coded colours in many places, best to use colours from the colours.txt file (should be pretty simple to find and replace these, some of them may need adding like the beautiful blue hold lines).
- Hold lines (and other items that naturally group together like taxiway centrelines and stand outlines) should ideally be sorted alphanumerically with a category heading. It's pretty easy to do this if you sort A-Z in the relevant folder in Google Earth before saving to .kmz and converting to .sct.
- Single line GEOs usually have in-line comments rather than above line (see how the existing one does it).
- "New Edinburgh SMR.kmz" isn't the best name for it since when there's the next new one it gets confusing, in my opinion you could just replace the existing one, or rename the new one to include the AIRAC cycle code.

Like I said, looks amazing, hope to be able to use it to control soon!

Peter

Robbo599 reviewed (changes requested):
please take a look at Peters's suggestions as I think they all need addressing as well.

SamLefevre commented:
Entirely forgot about this, will try and fix up tommorow

SamLefevre commented:
![image](https://user-images.githubusercontent.com/64741876/164513964-c320d882-c4d0-4dfd-aa92-41263a17ced0.png)

### Updated:

-Added colour definitions, couple hard colours still (0, for black in some lines), not sure how I would be able to mass change those without breaking everything, doing it manually will take a while
- More buildings around cargo area
- A/D taxi line stops at holding point
- Added H stand aprons
- Updated hold lines (now line up with text)
- Added disused areas/paths

luke11brown commented:
I'd suggest the following:
- Stand 21 label replaced with Stand 20 (20 is code C and therefore, used regularly)
- Stand 23 label removed
- Stand 24 label added

SamLefevre commented:
Tried to all things mentioned from above and from a closer reference image:
![image](https://user-images.githubusercontent.com/64741876/166454968-df842a6c-7c10-48f5-b08b-a5da3fd3aaca.png)

PLM1995 commented:
In colours.txt, I think the PH colours should come after the PG colours for alphabeticalness.

Peter
